src/Glpi/Features/PlanningEvent.php

```diff
@@ -1104,7 +1104,7 @@ public static function getSpecificValueToDisplay($field, $values, array $options
                     $users[] = sprintf(
                         '<a href="%s">%s</a>',
                         User::getFormURLWithID($user_id),
-                        getUserName($user_id, 1)
+                        \htmlspecialchars(getUserName($user_id, 1))
```

Comment:
`getUserName()` is already returning a HTML snippet when the `$link` param is no 0.
```suggestion
                        getUserName($user_id, 1)
```